### ISSUE TITLE
Using # character to comment rows collapses the commented row on next startup

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#770]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=735) by @jrantanen on 11 Mar 2011*

Following comment
| # My Keyword | Some argument | other argument | and | so on |
Is handled in next startup as
| # My Keyword Some argument other argument and so on |
which makes it impossible to uncomment the row easily.

Workaround is to use "Comment" keyword which is available from row's context menu. It should be available also in keyword editor that is currently blocked by the <a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=734" title="Row controls do not work from grid editor"> issue 734 </a>.

Currently Robot Frameworks parsing model handles commented rows incorrectly and that makes it impossible to RIDE to handle the commented rows correctly. After <a href="https://code.google.com/p/robotframework/issues/detail?id=576" rel="nofollow">https://code.google.com/p/robotframework/issues/detail?id=576</a> is fixed, this can be fixed in RIDE.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=735#c5) by @yanne on 22 Sep 2011*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=844" title="It should be possible to write comments in seperate columns."> Issue 844 </a> has been merged into this issue.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=735#c7) by andreas.brain on 12 Dec 2011*


The workaround with the Comment function won't work on test cases using a template, since the Comment keyword isn't considered there

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=735#c8) by @yanne on 4 Jan 2012*


This is now fixed so that Robot Framework parser preserves multi-cell comments, and it is possible to edit them in RIDE.
